packages/toolkit/connector-kit/src/index.ts

```diff
@@ -24,3 +24,5 @@ export const parseJson = (
     throw new ConnectorError(errorCode, errorPayload ?? jsonString);
   }
 };
+
+export const mockSmsVerificationCodeFileName = 'logto_mock_verification_code_record.txt';
```

Comment:
nit: not sure whether we should put this in a file with a filename like "constants.ts"?

Reply:
Let's do it when we have more constants to export